webserver/__init__.py

```diff
@@ -31,6 +33,9 @@ def create_app():
     from kafka_connection import init_kafka_connection
     init_kafka_connection(app.config['KAFKA_CONNECT'])
 
+    # Redis connection
+    app.redis = Redis()
```

Comment:
I mean is it actually a good practice to assign things directly to applications?